base_comment_template/tests/test_base_comment_template.py

```diff
@@ -1,37 +1,171 @@
-# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
-from odoo.tests.common import TransactionCase
+# Copyright 2020 NextERP Romania SRL
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from lxml import etree
 
+from odoo.tests import common
 
-class TestResPartner(TransactionCase):
-    def setUp(self):
-        super(TestResPartner, self).setUp()
-        self.template_id = self.env["base.comment.template"].create(
+from .fake_models import ResUsers, setup_test_model, teardown_test_model
+
+
+class TestCommentTemplate(common.SavepointCase):
+    at_install = False
+    post_install = True
```

Comment:
I'm not sure, should we use decorator instead ?

```suggestion
from odoo.tests import tagged


@tagged("post_install", "-at_install")
class TestCommentTemplate(common.SavepointCase):
```

(not sure you should apply this suggestion import wont be at the right place)

Reply:
This shouldn't be put at all. No need to execute at the end.